matter_server/server/device_controller.py

```diff
@@ -897,26 +918,6 @@ def reschedule() -> None:
             # https://github.com/project-chip/connectedhomeip/pull/26718
             reschedule()
 
-    @staticmethod
```

Comment:
this has nothing to do with logging while I do agree that it could be moved to a helper module.
Maybe update the PR description ?